docs/contributing/managing-pull-requests.md

```diff
@@ -0,0 +1,178 @@
+
+> 💡 If there isn't already an issue that describes the problem a PR is trying to solve, it can be helpful to open one. This helps in sharing context and can be very valuable in the future.
+
+## Verifying a Pull Request
+
+### General Guidelines
+
+Some general things to verify in a pull request are:
+
+- Links ought to be relative
+- Language ought to be inclusive and accessible
+
+### Type Specific Guidelines
+
+Each kind of PR also requires a different set of specific checks from us before they are merged in!
+
+Let's go over them real quick.
+
+#### Documentation
+
+We typically look for the following in PRs that add documentation:
+
+- Correctness — whether the added documentation is technically correct
+- Style — whether the written language follows our [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)
+
+#### Code
+
+For PRs that add code (whether a feature or fix), we look for the following:
+
+- Correctness — whether the code does what we think it does
+- Tests — when fixing a bug or adding a new feature, it can be very valuable to add tests. While we do merge some small PRs without them, more often than not, it's good to have tests asserting behaviour! This can be a combination of unit tests for the specific package, snapshot tests and end to end tests. The goal here is to ensure that something that is being fixed or added _remains_ fixed or working the way we expect it to. Good tests ensure this!
+- Code Quality — while it isn't good to nit pick, reasonable changes that improve readability are great to point out
+- Documentation in the package's README if you're adding something
+
+#### Starters or Site Showcase
+
+For PRs that add a site or a starter to the showcase, we ought to check:
+
+- We check if the site or starter is indeed a Gatsby site or starter respectively
+- Links — check if the links are working and accessible
+- Tags — ensure the tags are not arbitrary
+- Featured Status — for the site showcase, we like to default to _not_ featuring a site and typically someone from the Gatsby team features it later if they like!
+
+#### Blog posts
+
+For PRs that add a blog posts, we ought to check:
+
+- Correctness — whether the added documentation is technically correct
+- Style — whether the written language follows our [style guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)
+- Blog posts are similar to documentation in that they're long form writing, but also want to ensure that blog posts on [gatsbyjs.org](https://www.gatsbyjs.org) aren't purely promotional, spammy or inappropriate.
+
+## Automated Checks
+
+Our repository on [GitHub](https://github.com/gatsbyjs/gatsby) has several automated CI checks that are run automatically for all PRs. These include tests, linting and even preview builds [gatsbyjs.org](https://www.gatsbyjs.org).
```

Comment:
I'm not really sure what the link to .org is here for?

Reply:
I meant preview builds for .org but forgot to add the word _for_

Fixed!